docs/concepts/ipfs-implementations.md

```diff
@@ -12,26 +12,33 @@ Looking into implementing IPFS yourself?
 | ipfs-lite        | <https://github.com/hsanjuan/ipfs-lite>                                 | go                     | Minimal library oriented ipfs daemon building on the same blocks as Kubo but with a minimal glue layer.                 |
 | ipfs-nucleus     | <https://github.com/peergos/ipfs-nucleus/>                              | go                     | Minimal IPFS replacement for P2P IPLD apps.                                                                              |
-| ipget            | <https://github.com/ipfs/ipget>                                         | go                     | Minimal wget insipired tool to download files from IPFS nodes over bitswap.                                              |
 | iroh             | <https://github.com/n0-computer/iroh>                                   | rust                   | Extreme-Efficiency oriented IPFS implementation.                                                                         |
 | js-ipfs          | <https://github.com/ipfs/js-ipfs>                                       | javascript, typescript | Javascript implementation targeting nodejs and browsers.                                                                 |
 | Kubo             | <https://github.com/ipfs/kubo>                                          | go                     | Generalist daemon oriented IPFS implementation with an extensive HTTP API.                                               |
-| Linux2ipfs       | <https://github.com/Jorropo/linux2ipfs>                                 | go                     | Small pipeline and extreme-performance oriented implementation to upload files and deltas to pinning services very fast. |
 | Lotus            | <https://github.com/filecoin-project/lotus>                             | go                     | Filecoin node handling consensus, storage providing, making storage deals, importing data, ...                           |
+| whypfs           | <https://github.com/whyrusleeping/whypfs>                               | go                     | Daemon based on the same building blocks as Kubo but with some options tweaking for more performance.                    |
+
+## Legacy
+
+| Name             | URL                                                                     | Language(s)            | What it's trying to do                                                                                                   |
+|------------------|-------------------------------------------------------------------------|------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| auspinner        | <https://github.com/2color/auspinner>                                   | go                     | CLI tool to deal with the pinning service API and upload files through bitswap.                                          |
```

Comment:
On another thought, I think that because Auspinner still works and while not actively maintained still lives up to its promise so it may make sense to move it to Actively maintained.

However, this can be leading because its functionality is not on par with Kubo.

Therefore, If I think about this page more holistically, some ideas for better organisation come to mind:
- Adding an experimental column
- Adding a column for some of the IPFS subsystems, e.g. 
  - IPFS Gateway ('server' or 'client')
  - IPFS Pinning API Client
  - RPC API
  - DHT (client or server)
  - Level of IPLD support (not too sure about this one, but as an example, Auspinner takes CAR files which can be UnixFS or DAG-CBOR but it can't chunk and create UnixFS blocks for you)
  - PubSub support
  - Blockstore
  - IPNS support
  - Delegated Routing Spec Support
  - Long lived vs. ephemeral (js-ipfs, ipfs-lite)